webapp/src/components/Meeting.tsx

```diff
@@ -13,9 +14,17 @@ interface MeetingProps {
 
 const Meeting = ({ meetingId }: MeetingProps) => {
   const socket = useSocket();
+  const [meetingNotes, setMeetingNotes] = useState<MeetingNote[]>([]);
```

Comment:
Clarification on how the `<MeetingNote>` is being used here, what's the purpose of the first empty array bracket inside it?

Reply:
Any type with square brackets after it represents an array containing only members of that type. So, `MeetingNote[]` means "an array containing only `MeetingNote` objects". Does that answer your question?

Reply:
Yes, that answers my question, I was also looking at this [article](https://www.carlrippon.com/typed-usestate-with-typescript/) that seems to talk about the same thing about having a state that is strongly typed and how to deal with it.